inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -2931,37 +2930,66 @@ struct activation_test_params {
-#define CASE_ACTIVATION_3D_F32_3 {1, 16, 8, 8, 8}, data_types::f32, format::bfzyx, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_2 {1, 11, 3, 8}, data_types::f32, format::bfyx, activation_func::hard_sigmoid, {1.f, 1.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_3 {1, 4, 7, 12}, data_types::f32, format::bfyx, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_4 {1, 14, 8, 8}, data_types::f32, format::b_fs_yx_fsv16, activation_func::hyperbolic_tan, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_5 {1, 14, 8, 8}, data_types::f32, format::b_fs_yx_fsv16, activation_func::hard_sigmoid, {1.f, 1.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_6 {1, 14, 8, 8}, data_types::f32, format::b_fs_yx_fsv16, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_7 {1, 17, 31, 29}, data_types::f32, format::yxfb, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_8 {1, 17, 31, 29}, data_types::f32, format::byxf_af32, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_9 {1, 17, 31, 29}, data_types::f32, format::b_fs_yx_fsv32, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_F32_10 {1, 17, 31, 29}, data_types::f32, format::fyxb, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfyx
+#define CASE_ACTIVATION_3D_F32_0 {1, 16, 8, 4, 13}, data_types::f32, format::bfzyx, activation_func::hyperbolic_tan, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_1 {1, 16, 8, 8, 8}, data_types::f32, format::bfzyx, activation_func::hyperbolic_tan, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_2 {1, 16, 8, 8, 8}, data_types::f32, format::bfzyx, activation_func::hard_sigmoid, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_3 {1, 16, 8, 8, 8}, data_types::f32, format::bfzyx, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_4 {1, 17, 7, 7, 7}, data_types::f32, format::b_fs_zyx_fsv16, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_5 {1, 17, 7, 7, 7}, data_types::f32, format::b_fs_zyx_fsv32, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_6 {1, 17, 7, 7, 7}, data_types::f32, format::bs_fs_yx_bsv16_fsv16, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_7 {1, 17, 7, 7, 7}, data_types::f32, format::fs_b_yx_fsv32, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
+#define CASE_ACTIVATION_3D_F32_8 {1, 17, 7, 7, 7}, data_types::f32, format::fs_bs_yx_bsv4_fsv32, activation_func::relu, {0.f, 0.f}, data_types::f32, format::bfzyx
```

Comment:
Do we really need that much test cases for activation? The count of test cases here is pretty close to convolution's one...

E.g. CASE_ACTIVATION_3D_F32_{1,2,3} have exactly the same parameters except activation func, but different activation functions are supposed to be covered by activation primitive unit tests, not fusings tests.

Reply:
Test cases are now limited.